cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -375,4 +375,36 @@ private static List<String> validateComputeResources(ResourceRequirements resour
         }
         return errors;
     }
+
+    /**
+     * Create or update configuration value.
+     *
+     * @param config Configuration.
+     * @param key Property key.
+     * @param value Property value to add or replace.
+     */
+    public static void createOrUpdateConfigValue(AbstractConfiguration config, String key, String value) {
+        if (config != null && key != null && !key.isBlank() && value != null && !value.isBlank()) {
+            String existingConfig = config.getConfigOption(key);
+            if (existingConfig == null || !existingConfig.equals(value)) {
+                config.setConfigOption(key, value);
+            }
+        }
+    }
+
+    /**
+     * Create or add configuration list value.
+     *
+     * @param config Configuration.
+     * @param key Property key.
+     * @param value Property value to set or add.
+     */
+    public static void createOrAddConfigListValue(AbstractConfiguration config, String key, String value) {
+        if (config != null && key != null && !key.isBlank() && value != null && !value.isBlank()) {
+            String existingConfig = config.getConfigOption(key);
+            if (existingConfig == null || !existingConfig.contains(value)) {
+                config.setConfigOption(key, existingConfig == null ? value : existingConfig + "," + value);
+            }
+        }
+    }
```

Comment:
These both seem to be used only from one class so they should probably stay there.

Reply:
Moved to where you suggested.